### PR TITLE
Line highlighting

### PR DESCRIPTION
**Pull request by CatoTH**
The highlighting itself should work. Any recommendation about which material-modules to use for that tooltip-input to enter the line number?

### CODE REVIEW

**Comment by emanuelschuetze** [comment]
I would use a simple (number) input field with placeholder text (e.g. "go to line"); the input field should be styled like the form fields (just a buttom line). At the end of the field a icon button ("check" icon) should placed to confirm the input.
 
Still missing is the scrolling to highligted line.

@CatoTH Can you do this?

**Comment by CatoTH** [comment]
Done

**Comment by emanuelschuetze** [comment]
Some feedback after testing:
* catch error if input value is not in interval [1:max-line-number]
* improve ui: use a mac-icon-button with icon 'redo'. After clicking this button show input field. Increase the input field width (ca. 150px) to show full translated text. Add close icon to go back to hide input field and go back to mat-icon-button

**Comment by CatoTH** [comment]
Ok, I hope it's fine now. There are now two buttons next to each other once you're typing, which is a bit strange, but maybe isn't disturbing too much.
(Form validation in Angular's Material Design is really a pain, btw...)

File: client/src/app/site/motions/components/motion-detail/motion-detail.component.html [hunk 1]
```diff
@@ -360,6 +360,20 @@ <h4 translate>Origin</h4>
     >
         <!-- Line Number and Diff buttons -->
         <div *ngIf="!editMotion && !motion.isStatuteAmendment()" class="motion-text-controls">
+            <mat-form-field class="motion-goto-line" *ngIf="highlightedLineOpened">
+                <input type="number" [(ngModel)]="highlightedLineTyping" [ngModelOptions]="{standalone: true}" matInput
```
Comment (tsiegleauq) [inline client/src/app/site/motions/components/motion-detail/motion-detail.component.html]: please add the [osAutoFocus](https://github.com/OpenSlides/OpenSlides/blob/master/client/src/app/shared/directives/autofocus.directive.ts) directive
looks misaligned. Please use [prettier](https://prettier.io/playground/#N4Igxg9gdgLgprEAuEAeAllADgVxgAhgE8s4BeAHRChwFsAjOAJyvwG0AKKAcwFkIAJnAA2ASgC6lEAAt03acLnT4AgDKY4AFRKZurNj35DhAeSwx00AM6SqwKzACGUAY+HQ4SQkxxwAvqy0jjAAkth47MxMEEwAyk7wvMFg0sy2MkqK8irqUHBJMCnMgZhSAIysWMKOYHDSEMJCTFLAwPgA5ADiEIQ9innt+AA+3s5W1fD4fgEgAHwgADQgEOaWUFbIoI5M0QDuAArbCBsojgBuEOgCiyD0TDUA1nAwsVg1usgwPnBLyrTCAHVZPArG9arFjugLGcoURkOArBslpgrMwYPt7twgsgAGZuVFLABWVgAHgAhe5gJ4vRy0OC5OC4-E-EDEkmxXTCOAARRwEHgTOEBJAbyYqKY8Oc3Bw1QlSyw0VRAPuWHhCrg4rOjKWTDgAEccOhdRjHFjHILhajaOhPt8llZOTy+QKkHihSynPQAVcYNJkAAWJZfRzofrcADCEFo2JQGoArDccKjNI56Cc3cKFZgYN6BL7kGUAEwABmmQA) to align HTML

Comment (tsiegleauq) [inline client/src/app/site/motions/components/motion-detail/motion-detail.component.html]: localhost_4200_motions
the input field it too close to the motion text.
Opening the input field during mobile mode puts unwanted line breaks into the text.

**Pull request merged**